# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luafilesystem/no-compat-5.1.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN luafilesystem-1.1-vanilla/Makefile luafilesystem-1.1/Makefile
--- luafilesystem-1.1-vanilla/Makefile	2005-05-30 19:54:49.000000000 +0200
+++ luafilesystem-1.1/Makefile	2006-03-13 23:53:31.000000000 +0100
@@ -2,17 +2,14 @@
 
 T= lfs
 
-include ./config
-
 V= 1.1
 DIST_DIR= luafilesystem-$V
 TAR_FILE= $(DIST_DIR).tar.gz
 ZIP_FILE= $(DIST_DIR).zip
 LIBNAME= lib$T.$V.so
 
-COMPAT_O= $(COMPAT_DIR)/compat-5.1.o
 SRCS= src/$T.c
-OBJS= src/$T.o $(COMPAT_O)
+OBJS= src/$T.o
 
 
 lib: src/$(LIBNAME)
@@ -20,9 +17,6 @@
 src/$(LIBNAME): $(OBJS)
 	$(CC) $(CFLAGS) $(LIBS) $(LIB_OPTION) -o src/$(LIBNAME) $(OBJS)
 
-$(COMPAT_O): $(COMPAT_DIR)/compat-5.1.c
-	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
-
 install: src/$(LIBNAME)
 	mkdir -p $(LUA_LIBDIR)
 	cp src/$(LIBNAME) $(LUA_LIBDIR)
diff -ruN luafilesystem-1.1-vanilla/src/lfs.c luafilesystem-1.1/src/lfs.c
--- luafilesystem-1.1-vanilla/src/lfs.c	2005-05-20 20:32:19.000000000 +0200
+++ luafilesystem-1.1/src/lfs.c	2006-03-13 23:38:34.000000000 +0100
@@ -35,7 +35,6 @@
 #include "lua.h"
 #include "lauxlib.h"
 #include "lualib.h"
-#include "compat-5.1.h"
 
 #include "lfs.h"
 
